OutOfSchool/OutOfSchool.Encryption/Models/EnvelopedUserInfoResponse.cs

```diff
@@ -0,0 +1,10 @@
+namespace OutOfSchool.Encryption.Models;
+
+public class EnvelopedUserInfoResponse
+{
+    public string EncryptedUserInfo { get; set; }
+
+    public int Error { get; set; }
```

Comment:
Is this `int Error` something related to `HttpStatusCode` standard enum?

Reply:
model is copied from 3rd party authorization API and they do return `200 OK` & error code in body. Error code is their business logic code, not exactly HTTP code. We can check for error before calling this service, will remove this here and add check later in AuthServer.